configs/5.10/dynamic_springboard.patch

```diff
@@ -87,6 +73,14 @@ index d665eac45..f5038f4f0 100644
  	barrier();
  
  	return finish_task_switch(prev);
+@@ -4066,6 +4101,7 @@ pick_next_task(struct rq *rq, struct task_struct *prev, struct rq_flags *rf)
+  *
+  * WARNING: must be called with preemption disabled!
+  */
++__attribute__((optimize("no-omit-frame-pointer")))
```

Comment:
What's the difference between this and make prepare_task_switch noinline? And you should explain it in the commit log.